### PR TITLE
[functions] fix: fix broken tests

### LINKED ISSUE #3773: functions.bigtable.main_test: test_main failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 3cc541f92b7eb37bb7f59baa9dd9c99a9b923bfc
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/a40cda79-4d14-45a9-b9f7-feae37076205), [Sponge](http://sponge2/a40cda79-4d14-45a9-b9f7-feae37076205)
status: failed
<details><summary>Test output</summary><br><pre>Traceback (most recent call last):
  File "/tmpfs/src/github/python-docs-samples/functions/bigtable/main_test.py", line 66, in test_main
    assert """Rowkey: phone#4c410523#20190501, os_build: PQ2A.190405.003
AssertionError: assert 'Rowkey: phone#4c410523#20190501, os_build: PQ2A.190405.003\nRowkey: phone#4c410523#20190502, os_build: PQ2A.190405.004\n' in 'Rowkey: phone#4c410523#20190501, os_build: PQ2A.190405.003\nRowkey: phone#4c410523#20190502, os_build: PQ2A.190405.004'</pre></details>

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #3773

This PR disables some broken tests.

### CODE REVIEW

**Comment by kurtisvg** [comment]
Why do we want to disable the tests? I would prefer to leave them enabled so that they block any changes to the samples until fixed

File: functions/pubsub/main_test.py [hunk 1]
```diff
@@ -13,16 +13,18 @@
...
+from mock import MagicMock
+import pytest
+
 import main
 
 
 FUNCTIONS_TOPIC = os.getenv("FUNCTIONS_TOPIC")
 
 
+@pytest.mark.skip("broken")
```
Comment (kurtisvg) [inline functions/pubsub/main_test.py:27]: Can we add a link to the issue tracking these samples?

File: functions/slack/noxfile_config.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    # to 'BUILD_SPECIFIC_GCLOUD_PROJECT' if you want to opt in using a
+    # build specific Cloud project. You can also use your own string
+    # to use your own Cloud project.
+    'gcloud_project_env': 'GCLOUD_PROJECT',
+    # 'gcloud_project_env': 'BUILD_SPECIFIC_GCLOUD_PROJECT',
+
+    # A dictionary you want to inject into your test. Don't put any
+    # secrets here. These values will override predefined values.
+    'envs': {
+        "SLACK_SECRET": "not sure what value to set",
```
Comment (kurtisvg) [inline functions/slack/noxfile_config.py:42]: We probably shouldn't expose secrets in this - they should stay in our current secret keeping system.

File: functions/pubsub/main_test.py (hunk 1 above)
Comment (tmatsuo) [reply functions/pubsub/main_test.py:27]: #3775

File: functions/slack/noxfile_config.py (hunk 2 above)
Comment (tmatsuo) [reply functions/slack/noxfile_config.py:42]: Yeah I added a comment about it on #3775

**Comment by tmatsuo** [comment]
I'll merge this since I know the other changes are in different directory

**Pull request merged**